src/mui/input/ImageInput.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component, PropTypes } from 'react';
+import Dropzone from 'react-dropzone';
+
+const defaultStyle = {
+    dropZone: {
+        background: '#efefef',
+        cursor: 'pointer',
+        padding: '1rem',
+        textAlign: 'center',
+        color: '#999',
+    },
+    previewContainer: {
+        position: 'relative',
+        display: 'inline-block',
+    },
+};
+
+class ImageInput extends Component {
+    constructor(props) {
+        super(props);
+
+        let files = props.input.value || [];
+        if (!Array.isArray(files)) {
+            files = [files];
+        }
+
+        this.state = { files };
+    }
+
+    onDrop = (files) => {
+        const updatedFiles = [
+            ...this.state.files,
+            ...files.map(this.transformFile),
+        ];
+
+        this.setState({ files: updatedFiles });
+        this.props.input.onChange(files);
+    }
+
+    // turn a browser dropped file structure into expected structure
+    transformFile = (droppedFile) => {
+        const { source, title } = React.Children.toArray(this.props.children)[0].props;
+
+        const transformedFile = { ...droppedFile };
+        transformedFile[source] = droppedFile.preview;
+
+        if (title) {
+            transformedFile[title] = droppedFile.name;
+        }
+
+        return transformedFile;
+    };
+
+    label() {
+        if (this.props.multiple) {
+            return (
+                <p>Drop some files to upload, or click to select one.</p>
+            );
+        }
+
+        return (
+            <p>Drop a file to upload, or click to select it.</p>
+        );
+    }
+
+    render() {
+        const {
+            accept,
+            children,
+            disableClick,
+            maxSize,
+            minSize,
+            multiple,
+            style,
```

Comment:
`style` acts on the container element, see [SimpleForm](https://github.com/marmelab/admin-on-rest/blob/master/src/mui/form/SimpleForm.js#L14). You can't use it to style an element within your component, too.